Dockerfile

```diff
@@ -39,14 +39,14 @@ RUN wget -O /meta.yaml -q https://raw.githubusercontent.com/Open-MSS/MSS/${BRANC
    | sed -e "s/menuinst.*//" \
    | sed -e "s/.*://" > reqs.txt \
   && cat development.txt >> reqs.txt \
-  && echo pyvirtualdisplay >> reqs.txt \
   && mamba create -y -n mss-${BRANCH}-env --file reqs.txt \
   && mamba create -y -n mssenv --file reqs.txt \
-  && conda clean --all \
+  && mamba clean -y --all \
   && rm reqs.txt \
-  && cp /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh 
+  && cp /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh \
+  && cp /opt/conda/etc/profile.d/mamba.sh /etc/profile.d/mamba.sh
 
 # execute /etc/profile also in non-interactive use
-ENV BASH_ENV /etc/profile.d/conda.sh
+ENV BASH_ENV /etc/profile.d/mamba.sh
```

Comment:
when I install miniforge, I have to execute conda.sh and mamba.sh for everything to work...

Reply:
@joernu76 I tried locally with BRANCH set to develop and

```
docker run -it $(docker build -q .)
(base) root@e03e9a3fcf42:/# mamba activate mss-develop-env
```

Which failure have you seen?

Reply:
I *had* issues with an older version of the software. This was just a caveat. I approved the MR: